src/pages/ScoreReport.vue

```diff
@@ -577,11 +577,17 @@ const computeAssignmentAndRunData = computed(() => {
     const runsByTaskIdAcc = {};
 
     for (const { assignment, user } of assignmentData.value) {
+      // We can't garentee that the user's school is still in the current list of schools
+      //   So we'll find the school both in the 'all' list as well as assignment.readOrgs
+      const userSchools = user.schools?.all;
+      const assignmentOrgs = assignment.readOrgs.schools;
+      const matchingSchools = userSchools?.filter((schoolId) => assignmentOrgs.includes(schoolId));
+
       // for each row, compute: username, firstName, lastName, assessmentPID, grade, school, all the scores, and routeParams for report link
       const grade = user.studentData?.grade;
       // compute schoolName
       let schoolName = '';
-      const schoolId = user?.schools?.current[0];
+      const schoolId = matchingSchools[0];
```

Comment:
Is it possible that this could return the wrong school?

If a student has an old school in their `schools.all` array, and an assignment is also assigned to that old school?

Reply:
Yes, possibly, in the case that a user has switched schools within a district, and that the report is district level. 
This PR will resolve the issue for a majority of cases. A long term solution I've thought about is denormalizing the user's `minimalOrgs` into the assignment document, similar to how we do with the user data. However I feel as if I want to discuss that in a 16th minute when we're not doing sprint review.

Reply:
Could we see if `user?.schools?.current[0]` is in `assignment.readOrgs.schools` first? And if so, use that as the `schoolId`? And only if it isn't use the fallback that you have defined above where we compute matching schools?